solidity/test/random_beacon_operator/TestSlashing.js

```diff
@@ -157,18 +157,36 @@ describe('KeepRandomBeaconOperator/Slashing', function () {
     // There is only one active group in the system and that group did not
     // produce relay entry on time. Relay entry timeout is reported but since
     // there is no other group in the system, we do not retry with another
-    // group. This way, relay entry is timed out until someone requests for
-    // another entry and we need to make sure the group cannot be slashed more
-    // than one time.
+    // group. reportRelayEntryTimeout reverts so that the last group may not
+    // be slashed more than one time.
     it("reverts when already reported for the last active group", async () => {
       await time.advanceBlockTo(relayRequestStartBlock.addn(10))
       await operatorContract.reportRelayEntryTimeout({ from: tattletale })
       await expectRevert(
         operatorContract.reportRelayEntryTimeout({ from: tattletale }),
-        "Group has been already terminated"
+        "Entry did not time out"
       )
     })
 
+    // There is only one active group in the system and that group did not
+    // produce relay entry on time. Relay entry timeout is reported but since
+    // there is no other group in the system, we do not retry with another
+    // group. Entry is not marked as timed out to not block the beacon.
+    // With no groups, anyone can genesis again.
+    it("allows to genesis if reported for the last active group", async () => {
+      await time.advanceBlockTo(relayRequestStartBlock.addn(10))
+      await operatorContract.reportRelayEntryTimeout({ from: tattletale })
```

Comment:
Should we verify that no `RelayEntryRequested` is emitted here?

Reply:
I assume it's covered by `"should not be retried when there are no more active groups"` case in `TestRelayEntryTimeout.js`. And, to be honest, I think now this particular test should be moved to that file. It does not check anything about slashing. WDYT?

Reply:
Ah yes, makes sense. The move seems reasonable to me as well.